charts/consul/templates/mesh-gateway-deployment.yaml

```diff
@@ -262,7 +177,7 @@ spec:
           {{- end }}
       containers:
         - name: mesh-gateway
-          image: {{ .Values.global.imageEnvoy | quote }}
+          image: {{ .Values.global.imageConsulDataplane }}
```

Comment:
Might not be a bad idea to also have this templated with quotation marks as the Envoy image was. Just in case the image for Consul Dataplane gets to be something that YAML doesn't immediately recognize as a string.

Reply:
done!